star-gateway/WS_ARCHITECTURE.md

```
@@ -0,0 +1,1434 @@
+        s.hub.Broadcast(&starv1.STAREnvelope{
+            Payload: &starv1.STAREnvelope_System{System: req.SystemStatus},
+        })
+    }
+
+    return &gatewayv1.ForwardTelemetryResponse{}, nil
+}
+```
+---
+
+## 10. REST Endpoints
+```go
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)